wca-states.md

```diff
@@ -1,13 +1,13 @@
 # <wca-title>WCA List of Recognized Countries
 <wca-states>
 <version>Version: June 19, 2017
+  
+  
+The WCA recognizes a total of 199 states/regions:
```

Comment:
I would prefer to avoid "region", since we already use "region" to refer to explicitly refer to nations, continents, and the world: https://www.worldcubeassociation.org/regulations/#9i1

Could we just say "states?

Reply:
@lgarron If we used only one word, Chinese people would treat China and Taiwan as the same kind/level of a state (or some word else). This is like the current word "country".

Reply:
@lgarron please check the current email thread about this matter, were this specific wording is being discussed (we'd rather have the discussion at one single place).